### PR TITLE
Registration Page

### LINKED ISSUE #2: Registration Page [3pts]
Registration page:

- [x] Should be built with Devise.

### PR DESCRIPTION
**Pull request by andrianarivo**
Registration page:

- [x] Should be built with Devise.

### CODE REVIEW

**Review by rachaouldbabaali — CHANGES_REQUESTED** [review]
Hi @andrianarivo , 

Good job so far!
There are some issues that you still need to work on to go to the next task but you are almost there!


## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


 Happy coding!👏👏👏


------

File: app/views/devise/registrations/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      <%= f.email_field :email, autofocus: true, autocomplete: "email", class: 'form-control' %>
+    </div>
+
+    <div class="field mb-3">
+      <%= f.label :password, class: 'form-label' %>
+      <% if @minimum_password_length %>
+        <em>(<%= @minimum_password_length %> characters minimum)</em>
+      <% end %><br />
+      <%= f.password_field :password, autocomplete: "new-password", class: 'form-control' %>
+    </div>
```
Comment (rachaouldbabaali) [inline app/views/devise/registrations/new.html.erb]: Add "Name" Field to Registration Page: Currently, the registration page appears to lack a "Name" field. Including a "Name" field during user registration can provide a more personalized experience and improve user profiling. Ensure that this field is appropriately validated and stored in your user database.

**Comment by rachaouldbabaali** [comment]
image
edit recipe is not working well

**Comment by andrianarivo** [comment]
Thank you for your thorough review @rachaouldbabaali ! I'll make the changes and notify you soon

**Review by rachaouldbabaali — APPROVED** [review]
Hi @andrianarivo,

GIF

Your milestone is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
✅ You used Gitflow correctly
✅ No linter error
✅ Good commit messages
✅ Good descriptive PR

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

 Happy coding!👏👏👏

**Pull request merged**